docs/articles/tips-and-tricks/preferred-topology.rst

```diff
@@ -0,0 +1,69 @@
+Preferred topology
+===================
+
+If you want the flexibility to run tests against all topologies, but do not
+have the need to do it all the time. An example could be Nightly builds
+or and CI where a single topology is adequate coverage. This will reduce
```

Comment:
> where a single topology is adequate coverage

I would like to understand how this differs from `--mh-topology=<prefered topology>`. Could you explain that also in the documentation, not only in reply to my comment? I think that I get that but I'm not sure so more detailed description of the difference might be useful for others too.